proto/osmosis/accum/v1beta1/accum.proto

```diff
@@ -0,0 +1,30 @@
+syntax = "proto3";
+package osmosis.accum.v1beta1;
+
+import "gogoproto/gogo.proto";
+import "cosmos/base/v1beta1/coin.proto";
+
+option go_package = "github.com/osmosis-labs/osmosis/osmoutils/accum";
+
+message AccumulatorContent {
+  string accum_name = 1;
+  string accum_value = 2 [
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec",
+    (gogoproto.nullable) = false
+  ];
+}
+
+message Record {
+  string num_shares = 1 [
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int",
```

Comment:
Thoughts on making this decimal for future flexibility? If I'm not mistaken, even CL shares might be decimal as of right now

Reply:
Good point! I was thinking of shares as fungible LP shares (which have to be ints), but since in CL the definition of a share is a bit more specific Dec makes a lot of sense here